composer.json

```diff
@@ -35,7 +35,8 @@
     "jeroendesloovere/vcard": "v1.5",
     "browserstack/browserstack-local": "dev-master",
     "consolidation/robo": "^1.0.0",
-    "mockery/mockery": "^0.9.9"
+    "mockery/mockery": "^0.9.9",
+    "symfony/yaml": "^3.4"
```

Comment:
if it's an unused lib, can you remove please

Reply:
removed